packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -227,14 +234,25 @@ - (void)captureToFile:(NSString *)path result:(FlutterResult)result {
 
 - (void)setCaptureSessionPreset:(NSString *)resolutionPreset {
```

Comment:
(optional) Ah, okay. This is the exact same structure as the two new cases in Java. I think this would also be easier to read if it were split up like I mentioned above, but since this is existing code no pressure to refactor it.